TitanBar/TBresources.lua

```diff
@@ -62,6 +62,7 @@ _G.WalletItem = {
 	StarsofMerit = { Icon = 0x4115bea4 },
 	CentralGondorSilverPiece = { Icon = 0x41155957 },
 	GiftgiversBrand = { Icon = 0x4115c888 },
+	AshOfGorgoroth = { Icon = 0x411B91DD },
```

Comment:
How/where did you find this icon address?

Reply:
I used the addon ''ItemTreasury'' from Galuhad (http://www.lotrointerface.com/downloads/info870-ItemTreasury.html) to search for the Item.

There is no command to output the Icon ID but you can search the addon-file ''21_1_Items.lua'' for the Item ID string.

[1879352247]={[1]="Ash of Gorgoroth";[2]="This black ash contains remnants of the enchantment that once enveloped equipment from the plateau of Gorgoroth. Talented Elf-smiths can make use of this ash, and will accept it in exchange for valuable equipment.";[3]=178;[4]=4;[5]=0;[6]=false;[7]=false;[8]=0x411B91DD;[9]=0x410D461E;};

And [8] is the Icon ID

Reply:
Awesome!  Thanks!  I've got a lot of things planned that I'd like to add, and this will make some of them possible! 👍